DQMOffline/Trigger/plugins/JetMETMonitor.cc

```diff
@@ -0,0 +1,1237 @@
+     histtitle = "Neutral Em Energy Frac. vs HLT Jet pT";
+     bookJEFME(ibooker,hltjetJetFracME_NEEFVshltjetPt,histname,histtitle, met_binning_.nbins,met_binning_.xmin, met_binning_.xmax ,jet_enfrac_binning_.nbins, jet_enfrac_binning_.xmin, jet_enfrac_binning_.xmax);
+     setMETitle(hltjetJetFracME_NEEFVshltjetPt, "HLT Jet pT [GeV]","Fraction" );
+
+     histname = "hltjetJetFracME_CHEFVshltjetPt";
+     histtitle = "Charged Hadron Energy Frac. vs HLT Jet pT";
+     bookJEFME(ibooker,hltjetJetFracME_CHEFVshltjetPt,histname,histtitle, met_binning_.nbins,met_binning_.xmin, met_binning_.xmax ,jet_enfrac_binning_.nbins, jet_enfrac_binning_.xmin, jet_enfrac_binning_.xmax);
+     setMETitle(hltjetJetFracME_CHEFVshltjetPt, "HLT Jet pT [GeV]","Fraction" );
+
+     histname = "hltjetJetFracME_MuEFVshltjetPt";
+     histtitle = "Muon Energy Frac. vs HLT Jet pT";
+     bookJEFME(ibooker,hltjetJetFracME_MuEFVshltjetPt,histname,histtitle, met_binning_.nbins,met_binning_.xmin, met_binning_.xmax ,jet_enfrac_binning_.nbins, jet_enfrac_binning_.xmin, jet_enfrac_binning_.xmax);
+     setMETitle(hltjetJetFracME_MuEFVshltjetPt, "HLT Jet pT [GeV]","Fraction" );
+
+     histname = "hltjetJetFracME_CEEFVshltjetPt";
+     histtitle = "harged Em Energy Frac. vs HLT Jet pT";
+     bookJEFME(ibooker,hltjetJetFracME_CEEFVshltjetPt,histname,histtitle, met_binning_.nbins,met_binning_.xmin, met_binning_.xmax ,jet_enfrac_binning_.nbins, jet_enfrac_binning_.xmin, jet_enfrac_binning_.xmax);
+     setMETitle(hltjetJetFracME_CEEFVshltjetPt, "HLT Jet pT [GeV]","Fraction" );
+  }
+  // Initialize the GenericTriggerEventFlag
+  if ( num_genTriggerEventFlag_ && num_genTriggerEventFlag_->on() ) num_genTriggerEventFlag_->initRun( iRun, iSetup );
+  if ( den_genTriggerEventFlag_ && den_genTriggerEventFlag_->on() ) den_genTriggerEventFlag_->initRun( iRun, iSetup );
+
+}
+
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "DataFormats/TrackerCommon/interface/TrackerTopology.h"
+#include "Geometry/Records/interface/TrackerTopologyRcd.h"
+#include "DataFormats/Math/interface/deltaR.h" // For Delta R
+void JetMETMonitor::analyze(edm::Event const& iEvent, edm::EventSetup const& iSetup)  {
+  // Filter out events if Trigger Filtering is requested
+  if (den_genTriggerEventFlag_->on() && ! den_genTriggerEventFlag_->accept( iEvent, iSetup) ) return;
+
+  // PF Jet Trigger //
+  if (isPFJetTrig) {
+     edm::Handle<reco::PFJetCollection> jetHandle;
+     iEvent.getByToken( pfjetToken_, jetHandle );
+     std::vector<reco::PFJet> jets;
+     if ( int(jetHandle->size()) < njets_ ) return;
+     for ( auto const & jet : *jetHandle ) {
+       jets.push_back(jet);
+     }
+     if ( int(jets.size()) < njets_ ) return;
+     // HLT Jet Selection
+     edm::Handle<reco::PFJetCollection> hltjetHandle;
+     iEvent.getByToken( hltpfjetToken_, hltjetHandle );
+     std::vector<reco::PFJet> hlt_jets;
```

Comment:
To make efficiency plots of Jet, we defined numerator as match offilne jet with hltjet. 
So I used hlt jets. 
Denominator : Offline Jet > 20 GeV
Numerator : Offline Jet (matched hltjet , dR < 0.2 ).